src/pages/category/Category.js

```diff
@@ -1,23 +1,29 @@
+import { connect } from 'react-redux';
 import { useState, useEffect } from 'react';
+import { setCategory } from '../../store/categories';
 import { getCategoryService } from '../../services/categoriesService';
 import './category.scss';
 
-
-export const Category = ({ match }) => {
-  const [category, setCategory] = useState({});
+export const CategoryComponent = ({ match, dispatch }) => {
+  const [categoryState, setCategoryState] = useState({});
```

Comment:
According to [redux principles](https://redux.js.org/introduction/three-principles) you have to save data as a part of application model in a one place. So, the category should be stored in a redux. Also initial state for it is null (or undefined)